packages/sdk-components-react-radix/src/collapsible.tsx

```diff
@@ -35,3 +40,70 @@ export const CollapsibleTrigger = forwardRef<
 export const CollapsibleContent: ForwardRefExoticComponent<
   Omit<ComponentPropsWithRef<typeof Content>, "asChild">
 > = Content;
+
+/* BUILDER HOOKS */
+
+const namespace = "@webstudio-is/sdk-components-react-radix";
+
+const onCollapsibleContentMatched = (
```

Comment:
Lets add comment:
```
For each CollapsibleContent component within the selection, we identify its closest parent Collapsible component and execute the associated callback function.
```

Reply:
Probably function name like - matchClosestParents would be better

Reply:
matchClosestParents would not work because here we cover the case with nested collapsibles so we need to continue lookup after finding parent.